ws-client/src/client.rs

```diff
@@ -374,8 +411,50 @@ async fn background_task(
 					}
 				}
 			}
-			Either::Right((Some(Ok(jsonrpc::Response::Batch(_responses))), _)) => {
-				log::warn!("Ignore batch response not supported, #103");
+			Either::Right((Some(Ok(jsonrpc::Response::Batch(batch))), _)) => {
+				let mut digest = Vec::with_capacity(batch.len());
+				let mut ordered_responses = vec![JsonValue::Null; batch.len()];
+				let mut rps_unordered: Vec<_> = Vec::with_capacity(batch.len());
+
+				for rp in batch {
+					let id = match rp.id().as_number().copied() {
+						Some(id) => id,
+						None => {
+							let _ = front_error.send(Error::InvalidRequestId);
+							return;
+						}
+					};
+					let rp: Result<JsonValue, Error> = rp.try_into().map_err(Error::Request);
+					let rp = match rp {
+						Ok(rp) => rp,
+						Err(err) => {
+							let _ = front_error.send(err);
+							return;
+						}
+					};
+					digest.push(id);
+					rps_unordered.push((id, rp));
+				}
+
+				digest.sort();
```

Comment:
Github Actions says "used sort instead of sort_unstable to sort primitive type `u64`" here. Do you see those warnings too or is it just me?